### DISCUSSION TITLE
Toggle what libraries are displayed on the Home screen sections

### DISCUSSION THREAD

**Discussion by kayhos** [gotson/komga#1634] [Ideas]
This is a redo of #229 and #600, as they have been locked down, I am creating a new item.

I would like to have a way to exclude libraries from being displayed on the Home screen. This feature would be pertinent to regular users and administrators as sometime people want to have access to libraries but not necessarily have updates for them.

Multiple use cases could cause someone to want exclude libraries:
* A library in a language you do not understand.
* NSFW libraries
* Libraries that have content that is not pertinent or interesting to you.
* Reduce the content shown on the home page on very big Komga servers.

Thank you very much for developing Komga, it's a great software and it is the closest to my ideal feature set.

**Comment by gotson** [comment]
Duplicate of #1560
